app/components/input-field.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="input-field">
+  <label for={{@for}}>{{@label}}:</label>
+  <input
+    id={{@id}}
+    type={{@type}}
+    value={{@value}}
+    placeholder={{@placeholder}}
+    disabled={{@disabled}}
+    class={{if @showError 'input error-border' 'input'}}
```

Comment:
Can be updated to:
```
class="input {{if @showError "error-border"}}"
```
Helps improve readability a bit.

Refer https://guides.emberjs.com/release/components/conditional-content/#toc_inline-if

Also, "input" class on `<input>`?

Reply:
Makes sense! Thanks!